packages/data-context/src/sources/UtilDataSource.ts

```diff
@@ -18,4 +19,8 @@ export class UtilDataSource {
     // which is what will be used here
     return fetch(input, { agent, ...init })
   }
+
+  isDependencyInstalled (dependency: Cypress.CypressComponentDependency, projectPath: string) {
+    return isDependencyInstalled(dependency, projectPath)
```

Comment:
Nice to see we could leverage `isDependencyInstalled` making this PR pretty straight forward 👍